pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
+        <version>21</version>
+    </parent>
+
+    <groupId>org.camel.quarkus</groupId>
+    <artifactId>quarkus-camel-parent</artifactId>
+    <version>999-SNAPSHOT</version>
```

Comment:
Wondering if we should use a better version strategy rather than 999-SNAPSHOT

//cc @davsclaus

Reply:
I do not have a strong opinion on this. This is taken from Quarkus. IIRC their reasoning is that 999-SNAPSHOT will be always considered (by Maven) newer than any other version. Not sure this is motivated by CI or desktop development.

Reply:
Maybe one more advantage is that using single 999-SNAPSHOT version forever reduces the amount of clutter in the local Maven repo. This may save some space if the project is large.

A clear disadvantage is that this makes parallel working in older braches hard. Switching between 0.x and 1.x when reproducing bugs would always require full rebuild.

So I am quite ready to switch to 0.0.1-SNAPSHOT if you find it less controversial.

Reply:
I don't have any strong opinion either, I would go for x.y.z-SNAPSHOT just for consistency with other camel projects

Reply:
0ca5a6f has 0.0.1-SNAPSHOT